OloEditor/assets/models/Sponza/README.md

```
@@ -0,0 +1,92 @@
+Screenshot from the I3D paper
+http://crytek.com/sites/default/files/20100301_lpv.pdf
+```
+
+
+## Legal
+
+&copy; 2016, Crytek. [Cryengine Limited License Agreement](https://www.cryengine.com/ce-terms)
+
+ - Crytek for Everything
+
+#### Assembled by modelmetadata
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)